quickstarts/microsoft.network/secure-virtual-wan/azuredeploy.json

```diff
@@ -0,0 +1,1222 @@
+            "minValue": 1,
+            "maxValue": 10,
+            "metadata": {
+                "description": "Scale Units for Express Route Gateway in the second Hub"
+            }
+        },
+        "Hub1_P2SvpnGatewayScaleUnit": {
+            "defaultValue": 1,
+            "minValue": 1,
+            "maxValue": 200,
+            "type": "Int",
+            "metadata": {
+                "description": "Scale Units for Point-to-Site (P2S) VPN Gateway in the first Hub"
+            }
+        },
+        "Hub2_P2SvpnGatewayScaleUnit": {
+            "defaultValue": 1,
+            "minValue": 1,
+            "maxValue": 200,
+            "type": "Int",
+            "metadata": {
+                "description": "Scale Units for Point-to-Site (P2S) VPN Gateway in the second Hub"
+            }
+        },
+        "vmAdminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "Username for the Virtual Machine."
+            }
+        },
+        "vmAdminPassword": {
+            "type": "secureString",
+            "metadata": {
+                "description": "Password for the Virtual Machine."
+            }
+        },
+        "OSVersion": {
+            "type": "string",
+            "defaultValue": "2019-Datacenter",
+            "allowedValues": [
+                "2008-R2-SP1",
+                "2012-Datacenter",
+                "2012-R2-Datacenter",
+                "2016-Nano-Server",
+                "2016-Datacenter-with-Containers",
+                "2016-Datacenter",
+                "2019-Datacenter",
+                "2019-Datacenter-Core",
+                "2019-Datacenter-Core-smalldisk",
+                "2019-Datacenter-Core-with-Containers",
+                "2019-Datacenter-Core-with-Containers-smalldisk",
+                "2019-Datacenter-smalldisk",
+                "2019-Datacenter-with-Containers",
+                "2019-Datacenter-with-Containers-smalldisk"
+            ],
+            "metadata": {
+                "description": "The Windows version for the VM. This will pick a fully patched image of this given Windows version."
+            }
+        },
+        "vmSize": {
+            "type": "string",
+            "defaultValue": "Standard_B2s",
+            "metadata": {
+                "description": "Size of the virtual machine."
+            }
+        },
+        "vmNamePrefix": {
+            "type": "string",
+            "defaultValue": "VMVWAN",
+            "metadata": {
+                "description": "Name of the virtual machine."
+            }
+        },
+        "aadTenantId": {
+            "type": "string",
+            "defaultValue": "[subscription().tenantId]",
+            "metadata": {
+                "description": "Azure Active Directory Tenant Id"
+            }
+        }
+    },
+    "variables": {
+        "vwan_cfg": {
+            "type": "Standard"
+        },
+        "virtual_hub1_cfg": {
```

Comment:
for the cases where we have 4 resources, use arrays for the names/properties in the var section and then use a copy loop on the resource declarations below.